src/controllers/user/me.rs

```diff
@@ -8,7 +8,7 @@ use util::bad_request;
 
 use models::{Email, Follow, NewEmail, User, Version};
 use schema::{crates, emails, follows, users, versions};
-use views::{EncodablePrivateUser, EncodableVersion};
+use views::{EncodableMe, EncodableVersion};
```

Comment:
There are a few changes in a few commits (like this change to extract `EncodableMe` in a commit about moving `MockUserSession` to `util::TestApp`) that don't go with their commit's message, but they're pretty small so I'm not going to make you fix them. Just wanted to state for the record that you didn't get anything in without me noticing ;P

Reply:
Ha ha, yeah.  In general, I think there are more places we can extract view types in the main app and use the same types in the tests.  Maybe there is a reason we would want to have separate but identical types in the test, but I can't really think of problems that would help catch.